Packs/SecurityScorecard/Layouts/layoutscontainer-SecurityScorecard_Alert_Layout.json

```diff
@@ -313,46 +313,6 @@
                         "hideName": false,
                         "i": "caseinfoid-d1958ba0-dca7-11eb-b7f7-87923feabb22",
                         "items": [
-                            {
```

Comment:
Why would we want to remove these?

Reply:
Those fieldId were removed in the past I see.
they are `securityscorecardgrade`  `securityscorecardscore`  `securityscorecardfactor` `securityscorecardscoreimpact`